SQL/database_changelog.txt

```diff
@@ -1,3 +1,13 @@
+30 January 2017, by Lzimann
+
+Modified table 'death', adding the columns 'mapname' and 'server'.
+
+ALTER TABLE `death` ADD COLUMN `mapname` TEXT NOT NULL AFTER `coord`, ADD COLUMN `server` TEXT NOT NULL AFTER `mapname`
```

Comment:
Ya no i'm stupid, for text columns it just defaults to a blank string when adding the column. ignore this.